### PR TITLE
AAP-30808: Upgrade/migration guide asciidoc conversion 

### PR DESCRIPTION
**Pull request by hherbly**
See [AAP-30808](https://issues.redhat.com/browse/AAP-30808) for more info.

### CODE REVIEW

File: downstream/modules/platform/con-aap-upgrade-planning.adoc [hunk 1]
```diff
@@ -7,34 +7,20 @@
...
-[role="_additional-resources"]
-.Additional resources
-* <<editing-inventory-file-for-updates_{context}, Setting up the inventory file >>
-
-[discrete]
-== {EDAcontroller}
-//ATTENTION: Remove this section for EDA 1.0.4; customers will no longer need to perform deactivation because services will be automatically restored after upgrade and migration. 
+* link:{URLCentralAuth}/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching a subscription]
+* xref:platform/con-backup-aap.adoc[Backup and restore]
+* link:{URLControllerAdminGuide}/controller-clustering[Clustering]
```
Comment (lmalivert) [inline downstream/modules/platform/con-aap-upgrade-planning.adoc:25]: A fourth bullet is missing, "Planning your installation"

File: downstream/modules/platform/con-aap-upgrade-planning.adoc [hunk 2]
```diff
@@ -7,34 +7,20 @@
...
+* See link:{URLPlanningGuide}/platform-system-requirements[System requirements] in the {TitlePlanningGuide} guide to ensure you have the topologies that fit your use case. 
++
+[NOTE]
+====
+2.4 to 2.5 upgrades now include link:{URLPlanningGuide}/ref-aap-components#con-about-platform-gateway_planning[Platform gateway]. Ensure you review the 2.5 link:{URLPlanningGuide}/ref-network-ports-protocols_planning[Network ports and protocols] for architectural changes.
+====
++
+* Ensure you have a valid subscription from a previous version. You must provide your credentials or a link:https://access.redhat.com/articles/5807761[subscriptions manifest] upon upgrading to the latest version of {PlatformNameShort}.
+* Clustered upgrades require special attention to instance and instance groups before upgrading. Ensure you capture your inventory or instance group details before upgrading.
+* Upgrading 2.4 {EDAName} is not supported.[ADD FURTHER TEXT HERE]
```
Comment (lmalivert) [inline downstream/modules/platform/con-aap-upgrade-planning.adoc:19]: @hherbly A reminder to add the additional text.

File: downstream/modules/platform/con-backup-aap.adoc [hunk 3]
```diff
@@ -2,16 +2,18 @@
...
+
+. Navigate to your {PlatformNameShort} installation directory.
 . Run the `./setup.sh` script following the example below:
 +
 ----
 $ ./setup.sh -e ‘backup_dir=/ansible/mybackup’ -e ‘use_compression=True’ @credentials.yml -b <1><2>
 ----
 <1> `backup_dir` specifies a directory to save your backup to.
-<2> `@credentials.yml` passes the password variables and their values encrypted via `ansible-vault`.
+<2> `@credentials.yml` passes the password variables and their values encrypted via `ansible-vault`. [DOUBLE CHECK FOR NEW CONTENT]
```
Comment (lmalivert) [inline downstream/modules/platform/con-backup-aap.adoc:16]: I'll make sure to get confirmation on the removal of this bullet point.

File: downstream/modules/platform/con-aap-upgrade-planning.adoc (hunk 1 above)
Comment (hherbly) [reply downstream/modules/platform/con-aap-upgrade-planning.adoc:25]: Good catch! Should be fixed now.

File: downstream/modules/platform/con-aap-upgrade-planning.adoc (hunk 2 above)
Comment (hherbly) [reply downstream/modules/platform/con-aap-upgrade-planning.adoc:19]: I definitely will. This is a spot where Satoe commented.

**Review by dcdacosta — APPROVED** [review]
With the plan to address outstanding issues in a new PR, this LGTM!

**Pull request merged**